### PR TITLE
Create measures components such as Form, Select, Simple View, combined component ModalWithForm

### PR DESCRIPTION
**Pull request by sashka2131**

### CODE REVIEW

File: src/app/domains/Measure/components/combined/MeasureModalWithForm/MeasureModalWithForm.template.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { useForm } from 'react-hook-form'
+import { Modal } from 'app/components/Lib/Modal'
+import { MeasureSimpleForm } from '../../forms'
+
+const MeasureModalWithForm = () => {
+  const [open, setOpen] = useState(false)
+  const measureForm = useForm()
+
+  const onSubmit = ({ measure }) => {
+    console.log(measure)
```
Comment (ARTfun00) [inline src/app/domains/Measure/components/combined/MeasureModalWithForm/MeasureModalWithForm.template.js:13]: Please, remove unnecessary logs

File: src/app/domains/Measure/components/select/MeasureSingleSelect/MeasureSingleSelect.template.js [hunk 2]
```diff
@@ -1,8 +1,25 @@
+import { TextField, MenuItem } from '@material-ui/core'
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+
 const MeasureSingleSelect = (props) => {
-  return <div>MeasureSingleSelect</div>
+  const { value, onChange } = props
+  const [currentMeasure, setCurrentMeasure] = useState(value)
+  const MEASURES = ['кг', 'г', 'л', 'мм', 'см', 'м']
```
Comment (ARTfun00) [inline src/app/domains/Measure/components/select/MeasureSingleSelect/MeasureSingleSelect.template.js:8]: Are u sure that we will use Russian language for measure variants ?)
Please, move this variable at least outside of the component scope...
Ideally, it should be in the separate singe file with constants for the app or module...

**Review by ARTfun00 — CHANGES_REQUESTED** [review]

File: src/app/constants/firebaseConfig.js [hunk 3]
```diff
@@ -1,12 +1,6 @@
...
-  https://expenses-app-development-9ba1c.web.app/
-
-
-production URL
-  https://expenses-app-production.web.app/
-
- */
+import 'firebase/auth'
+import 'firebase/database'
+import 'firebase/firestore'
```
Comment (GingerBoy14) [inline src/app/constants/firebaseConfig.js:3]: we will have separate firebase imports in services

File: src/app/domains/Measure/components/select/MeasureSingleSelect/MeasureSingleSelect.template.js [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+      onChange={(e) => setCurrentMeasure(e.target.value)}
+      value={currentMeasure}>
+      {MEASURES.map((item) => (
+        <MenuItem key={item} value={item}>
+          {item}
+        </MenuItem>
+      ))}
+    </TextField>
+  )
 }
```
Comment (GingerBoy14) [inline src/app/domains/Measure/components/select/MeasureSingleSelect/MeasureSingleSelect.template.js:21]: You can refactor this with new Select component

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**